README.md

```diff
@@ -114,11 +117,38 @@ Run the following command to start the docker container in interactive mode.
 docker run -ti --gpus all --rm --ipc=host colossalai bash
 ```
 
-## Contributing
 
-If you wish to contribute to this project, you can follow the guideline in [Contributing](./CONTRIBUTING.md)
+## Community
+
+Join the Colossal-AI community on [Forum](https://github.com/hpcaitech/ColossalAI/discussions),
+[Slack](https://join.slack.com/t/colossalaiworkspace/shared_invite/zt-z7b26eeb-CBp7jouvu~r0~lcFzX832w),
+and [WeChat](./docs/images/WeChat.png "qrcode") to share your suggestions, advice, and questions with our engineering team.
+
 
 
+## Contributing
+
+If you wish to contribute to this project, you can follow the guideline in [Contributing](./CONTRIBUTING.md).
+
+Thanks so much to all of our amazing contributors!
+
+<br>
+<a href="https://github.com/FrankLeeeee"><img src="https://avatars.githubusercontent.com/u/31818963?v=4" width="30px" /></a>
+<a href="https://github.com/ver217"><img src="https://avatars.githubusercontent.com/u/23111350?v=4" width="30px" /></a>
+<a href="https://github.com/kurisusnowdeng"><img src="https://avatars.githubusercontent.com/u/13151642?v=4" width="30px" /></a>
+<a href="https://github.com/binmakeswell"><img src="https://avatars.githubusercontent.com/u/61670638?v=4" width="30px" /></a>
+<a href="https://github.com/BoxiangW"><img src="https://avatars.githubusercontent.com/u/45734921?v=4" width="30px" /></a>
+<a href="https://github.com/1SAA"><img src="https://avatars.githubusercontent.com/u/72907851?v=4" width="30px" /></a>
+<a href="https://github.com/WANG-CR"><img src="https://avatars.githubusercontent.com/u/46049915?v=4" width="30px" /></a>
+<a href="https://github.com/feifeibear"><img src="https://avatars.githubusercontent.com/u/5706969?v=4" width="30px" /></a>
+<a href="https://github.com/Gy-Lu"><img src="https://avatars.githubusercontent.com/u/74758262?v=4" width="30px" /></a>
+<a href="https://github.com/DevinCheung"><img src="https://avatars.githubusercontent.com/u/22792747?v=4" width="30px" /></a>
+<a href="https://github.com/ProgrammerNeoo"><img src="https://avatars.githubusercontent.com/u/32625025?v=4" width="30px" /></a>
+<a href="https://github.com/Shenggan"><img src="https://avatars.githubusercontent.com/u/20657639?v=4" width="30px" /></a>
+<a href="https://github.com/xcetos"><img src="https://avatars.githubusercontent.com/u/45086830?v=4" width="30px" /></a>
+<a href="https://github.com/Sze-qq"><img src="https://avatars.githubusercontent.com/u/68757353?v=4" width="30px" /></a>
```

Comment:
Do not hardcode this. This list will grow and requires frequent maintenance if hardcoded. Try using some online API, e.g. [contrib.rocks](https://contrib.rocks/preview?repo=hpcaitech%2FColossalAI/preview?repo=hpcaitech%2FColossalAI). This will generate an image with the list of contributors automatically without manual synchronization.

Reply:
Yes, I know it can be generated an image. I just want to allow direct click to each contributor's home page, which needs a complex robot to generate a very long hardcode as some famous project. I just provide a temporary hardcode solution. Even if we use a figure to show it, we still need a robot to update it every day or with other commits to maintain randomly shuffled and latest.  I will try to update it with the figure solution if I have time.

Reply:
see this project for more reference.
https://github.com/taichi-dev/taichi

Reply:
Yes, I know it, it use a whole figure to show contributors, and use a robot to update it everyday.

Reply:
sounds good

Reply:
Yes, I know it, it uses a whole figure to show contributors, and uses a robot to update it every day.

Reply:
I can create a workflow for this today.

Reply:
Can you try this? https://github.com/hpcaitech/public_assets/blob/main/colossalai/img/contributor_avatar.png

Reply:
This workflow is run daily. If you wish to make it take effect immediately. You can run the workflow on main branch manually [here](https://github.com/hpcaitech/public_assets/actions/workflows/genereate_contributor_list.yml)

Reply:
Remember to use github raw content link, https://raw.githubusercontent.com/hpcaitech/public_assets/main/colossalai/img/contributor_avatar.png